Koronavilkku/Koronavilkku/ViewController/EndOfLifeViewController.swift

```diff
@@ -0,0 +1,85 @@
+import UIKit
+import Combine
+import SnapKit
+
+
+class EndOfLifeViewController: UIViewController {
+    
+    let radarImageView = UIImageView()
+    let logoImageView = UIImageView()
+    
+    let titleLabel = UILabel()
+    let textLabel = UILabel()
+    
+
+    var stackView = UIStackView()
+    
+    var radarImage = UIImage(named: "radar-off")
+    var logoImage = UIImage(named: "THL-logo")
+    
+    var titleText = Translation.EndOfLifeTitle.localized
+    var textLabelText = Translation.EndOfLifeMessage.localized
+    
+    var configurationTask: AnyCancellable?
```

Comment:
This is not used

Reply:
I've removed configurationTask, however, titleText and textLabelText are used while setting up title and description.